src/main/kotlin/com/nextstep/racingcar/domain/RacingGame.kt

```diff
@@ -0,0 +1,32 @@
+package com.nextstep.racingcar.domain
+
+import com.nextstep.racingcar.view.InputView
+import com.nextstep.racingcar.view.ResultView
+
+class RacingGame(private val numberGenerator: NumberGenerator, private val moveRule: MoveRule) {
+    private val inputView = InputView()
+    private val resultView = ResultView()
+
+    fun start() {
+        val numberOfCars = inputView.inputNumberOfCars()
+        val numberOfRounds = inputView.inputNumberOfRounds()
+
+        val cars = mutableListOf<Car>()
+        for (i in 1..numberOfCars) {
+            cars.add(Car())
+        }
+
```

Comment:
```suggestion
        val cars = (1..numberOfCars).map { Car() }
```
이렇게 코틀린의 확장함수를 활용해볼 수 있겠네요 :)